x11-misc/xflux/xflux-20130927.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Makes the color of your computer's display adapt to the time of the day"
+HOMEPAGE="https://justgetflux.com/"
+SRC_URI="amd64? ( https://justgetflux.com/linux/${PN}64.tgz -> ${PN}64-${PV}.tar.gz )
+	x86? ( https://justgetflux.com/linux/${PN}-pre.tgz -> ${P}.tar.gz )"
+
+KEYWORDS="-* ~amd64 ~x86"
+LICENSE="f.lux"
+SLOT="0"
+
+RESTRICT="bindist mirror strip"
```

Comment:
Why do you need to restrict stripping?

Reply:
Maybe that's my wrong interpretation, but should't binaries remain unchanged, so disallow strip? If that's wrong or sth, I can remove it.

Reply:
I haven't read the license thoroughly but at a first glance, I don't see anything that would prevent the user from stripping binaries.

Reply:
Seems you are right. I don't see this restriction neither. Dropped strip.